sig-data-science/charter.md

```diff
@@ -20,31 +20,33 @@ _Watch this [video](https://youtu.be/v4mur-_ywBM) for an in-depth discussion on
 
 #### Code, Binaries and Services
 
-* Publish and promote videos, blogs, talks, and tutorials on the tools, projects and best practices developed by the Data Science SIG subprojects.
+* Develop and incubate new AI applications and services
 
-* Develop demos, docs and notebooks to illustrate the data science capabilities of Operate First.
+* Leverage the data produced by the Operate First community cluster to provide AIOps tools and analyses to the community.
 
 * Curate and maintain data sets generated by the Operate First platform.
 
 * Experiment with and evaluate cloud native data science development tools and methods.
 
-* Leverage the data produced by the Operate First community cluster to provide AIOps tools and analyses to the community.
+* Publish and promote videos, blogs, talks, and tutorials on the tools, projects and best practices developed by the Data Science SIG subprojects.
```

Comment:
It is definitely correct to capitalize in proper names and titles, e.g. "Data Science SIG is a group of people interested in data science."